NumLib/NumericsConfig.h

```diff
@@ -12,6 +12,8 @@
 
 #include <type_traits>
 
+#include "MathLib/LinAlg/SparsityPattern.h"
```

Comment:
Why not move it to MathLib alongside with GlobalSetup? The two files only reference MathLib stuff, or am I wrong?